framework/include/base/FEProblem.h

```diff
@@ -324,13 +325,16 @@ class FEProblem :
    */
   void outputStep(ExecFlagType type);
 
+  ///@{
   /**
    * Ability to enable/disable all output calls
    *
-   * This is needed by RattleSNake/YAK to disable output because of the Yo Dawg executioners calling
+   * This is needed by Multiapps and RattleSNake/YAK to disable output because of the Yo Dawg executioners calling
```

Comment:
I want them for when I need to change something and I am trying to remember
why something was utilized, but if really matters to you I can pull them
out.

On Wed, Feb 11, 2015 at 4:06 PM, Cody Permann <notifications@github.com>
wrote:

> In framework/include/base/FEProblem.h
> <https://github.com/idaholab/moose/pull/4663#discussion_r24543334>:
>
> >    /**
> >     * Ability to enable/disable all output calls
> >     *
> > -   * This is needed by RattleSNake/YAK to disable output because of the Yo Dawg executioners calling
> > +   * This is needed by Multiapps and RattleSNake/YAK to disable output because of the Yo Dawg executioners calling
>
> Do we want comments referring to internal applications, especially since
> this is part of the doxygen doco?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/4663/files#r24543334>.
>